apps/teams-test-app/src/components/PagesAPIs.tsx

```diff
@@ -121,6 +122,28 @@ const ReturnFocus = (): React.ReactElement =>
     },
   });
 
+const RegisterFocusEnterHandler = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'registerFocusEnterHandler',
+    title: 'Register On Focus Enter Handler',
+    onClick: {
+      withPromise: async setResult => {
```

Comment:
These look exactly the same. you don't really need both.

Reply:
Sure, I can keep only one. btw, I copied it from teamsCoreAPI, where it was added by PR#854 (https://github.com/OfficeDev/microsoft-teams-library-js/pull/854/files)